packages/forma-36-react-components/src/components/Switch/Switch.stories.tsx

```diff
@@ -5,7 +5,7 @@ import SectionHeading from '../Typography/SectionHeading';
 import Flex from '../Flex/Flex';
 
 export default {
-  title: 'Components/Switch',
+  title: 'Forms/Switch',
```

Comment:
I'm not sure about this one 🤔

Reply:
I'm not sure also
I added it because it's actually a checkbox if you think about and I was ~copying~ inspired by CircuitUI

I will remove it for now. We can revisit this later